packetbeat/beater/install_npcap_test.go

```diff
@@ -213,7 +213,7 @@ func TestCanInstallNpcap(t *testing.T) {
 				BeatConfig: cfg,
 				Manager:    boolManager{managed: test.managed},
 			}
-			got, err := canInstallNpcap(b)
+			got, err := canInstallNpcap(b, b.BeatConfig)
```

Comment:
Minor thought : Can we pass `cfg` instead? Just to avoid the minor confusion that https://github.com/elastic/beats/pull/35935/files#diff-69402794a921b13ab8019aa16a00528596ce02e9ab1b40d7d266c44cc0bed664L122 has changed `b.BeatConfig` to `rawCfg`

Reply:
Yeah, OK. It shouldn't be any more confusing than it already is.